test/end-to-end/helpers/index.js

```diff
@@ -70,8 +70,11 @@ helpers.runAuthFlow = async function(authUrl) {
     .forBrowser('firefox')
     .build();
 
-  // OEM Selector
   await driver.get(authUrl);
+  // Preamble
+  await driver.findElement(By.css('button[id="continue-button"]')).click();
```

Comment:
you can actually find by id 
```suggestion
  await driver.findElement(By.id('continue-button')).click();
```